File: app/src/main/java/br/com/jonathanzanella/myexpenses/resume/MonthlyPagerAdapterHelper.java
package br.com.jonathanzanella.myexpenses.resume;

import org.joda.time.DateTime;

import java.text.SimpleDateFormat;
import java.util.Locale;

class MonthlyPagerAdapterHelper {
	private static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat("MMM/yy", Locale.getDefault());

	String formatMonthForView(DateTime month) {
		return SIMPLE_DATE_FORMAT.format(month.toDate());
	}
}

File: app/src/main/kotlin/br/com/jonathanzanella/myexpenses/resume/MonthlyPagerAdapterHelper.kt
package br.com.jonathanzanella.myexpenses.resume

import org.joda.time.DateTime

import java.text.SimpleDateFormat
import java.util.Locale

class MonthlyPagerAdapterHelper {

    fun formatMonthForView(month: DateTime): String {
        synchronized(this) {
            return SIMPLE_DATE_FORMAT.format(month.toDate())
        }
    }

    companion object {
        private val SIMPLE_DATE_FORMAT = SimpleDateFormat("MMM/yy", Locale.getDefault())
    }
}